guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -1,44 +1,34 @@
 [id="Configuring_a_Host_for_Registration_{context}"]
 = Configuring a Host for Registration
 
-{RHEL} virtual machines register to Customer Portal Subscription Management by default.
-You must update each virtual machine configuration so that they receive updates from the correct {Project} Server or {SmartProxyServer}.
+{EL} virtual machines register to Customer Portal Subscription Management by default.
```

Comment:
That is probably RHEL-specific.

For Satellite, you can use "RHEL"; for everything else, you can say "Register your hosts to {ProjectServer} or {SmartProxyServer} to ..." as a single sentence. Feel free to extend/reword.

Reply:
Not even mentioning that the sentence is useless in this place :D I'll just remove it.

Reply:
Or maybe not. I'll just rewrite it.